app/presenters/publishing_api_presenters/detailed_guide.rb

```diff
@@ -0,0 +1,71 @@
+require_relative "../publishing_api_presenters"
+
+class PublishingApiPresenters::DetailedGuide < PublishingApiPresenters::Edition
+  def links
+    extract_links([
+      :lead_organisations
+    ]).merge(
+      related_guides: related_guides,
+      related_mainstream: related_mainstream
+    )
+  end
+
+private
+
+  def document_format
+    "detailed_guide"
+  end
+
+  def details
+    super.merge(
+      body: body,
+      first_public_at: first_public_at,
+      change_history: item.change_history.as_json,
+      related_mainstream_content: related_mainstream,
+      political: item.political?,
+      government: government
+    )
+  end
+
+  def body
+    Whitehall::GovspeakRenderer.new.govspeak_edition_to_html(item)
```

Comment:
I think the `body` presenter needs to use `govspeak_with_attachments_to_html` so that embedded attachments render correctly.